packages/typespec-ts/test/integration/azureClientGeneratorCoreAccess.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { assert } from "chai";
+import AccessClientFactory, {
+  AccessClient
+} from "./generated/azure/clientGeneratorCore/access/src/index.js";
+import { matrix } from "../util/matrix.js";
+
+describe("Access Client", () => {
+  let client: AccessClient;
+
+  interface UrlDetail {
+    type: string;
+    value: string;
+  }
+
+  const testedUrl: UrlDetail[] = [
+    {
+      type: "publicOperation",
+      value: "noDecoratorInPublic"
+    },
+    {
+      type: "publicOperation",
+      value: "publicDecoratorInPublic"
+    },
+    {
+      type: "internalOperation",
+      value: "noDecoratorInInternal"
+    },
+    {
+      type: "internalOperation",
+      value: "internalDecoratorInInternal"
+    },
+    {
+      type: "internalOperation",
+      value: "publicDecoratorInInternal"
+    },
+    {
+      type: "sharedModelInOperation",
+      value: "public"
+    },
+    { type: "sharedModelInOperation", value: "internal" }
+  ];
+
+  beforeEach(() => {
+    client = AccessClientFactory({
+      allowInsecureConnection: true,
+      retryOptions: {
+        maxRetries: 0
+      }
+    });
+  });
+
+  matrix([testedUrl], async (params: UrlDetail) => {
+    it(`should access ${params.value} in ${params.type}`, async () => {
+      try {
+        const result = await client
+          .path(
+            `/azure/client-generator-core/access/${params.type}/${params.value}` as any
+          )
+          .get({ queryParameters: { name: "myname" } });
+        assert.strictEqual(result.status, "200");
+        assert.strictEqual(
+          JSON.stringify(result.body),
```

Comment:
Why do we need the stringify here? I think the correct way is directly using `.` for any property e.g `result.body.name`.

Reply:
I have updated the code as you said and will do the same in the future.